src/log.c

```diff
@@ -16,16 +16,26 @@
 #include "log.h"
 
 #define MAX_BUFFER_LEN 2048
-
-char log_level_str[][30] = {
-    "\033[1;38;5;4mDEBUG",
-    "\033[1;38;5;82mINFO",
-    "\033[1;38;5;11mWARN",
-    "\033[1;38;5;9mERROR",
-    "\033[1;38;5;9mFATAL"
+#define COLOR_TERM "\033[0m"
+
+char *log_level_str[2][30] = {
```

Comment:
I believe `*log_level_str[2][5]` is enough in this case and this variable could be set to constant.